### PR TITLE
Update cherry_pick_pull.sh

### PR DESCRIPTION
**Pull request by Kartik494**
#### What type of PR is this?
/kind bug
/kind cleanup

#### What this PR does / why we need it:

This PR improve message statement in `cherry_pick_pull.sh` , previous was confusing i.e not specify exact behaviour  
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by Kartik494** [comment]
Hi @justaugustus @SataQiu Could you please take a look here.
Thanks!

File: hack/cherry_pick_pull.sh [hunk 1]
```diff
@@ -180,7 +180,7 @@ for pull in "${PULLS[@]}"; do
       (git status --porcelain | grep ^U) || echo "!!! None. Did you git am --continue?"
       echo
       echo "+++ Please resolve the conflicts in another window (and remember to 'git add / git am --continue')"
-      read -p "+++ Proceed (anything but 'y' aborts the cherry-pick)? [y/n] " -r
+      read -p "+++ Proceed (anything apart from 'y' aborts the cherry-pick)? [y/n] " -r
```
Comment (dims) [inline hack/cherry_pick_pull.sh:183]: can we do `other than` instead of `apart from`?

Comment (Kartik494) [reply hack/cherry_pick_pull.sh]: Hi @dims Thanks for your suggestions. I have addressed the comments PTAL!!

**Pull request merged**